docs/source/using/addons.adoc

```diff
@@ -180,6 +180,19 @@ Multiple variables need to be comma separated.
 # Required-Vars: PROJECT_USER
 ----
 
+In case, you want to have a default value for a variable, you can achieve that with _Var-Defaults_ metadata header.
```

Comment:
```adoc
Default values for variables may be achieved using the _Var-Defaults_ metadata header.
```

Reply:
> Default values for variables may be achieved using the _Var-Defaults_ metadata header.

@robin-owen , `may be` might not be good phrase here since we want to inform user it is the way to add default value.

Fixed with `Default values for variables can be achieved using the _Var-Defaults_ metadata header.`